dune-workspace.template

```diff
@@ -8,4 +8,4 @@
 (context (opam (switch 4.08.1)))
 (context (opam (switch 4.09.0)))
 (context (opam (switch 4.10.0) (merlin)))
-(context (opam (switch ocaml-variants.4.11.0+beta2)))
+(context (opam (switch 4.11.0)))
```

Comment:
Can we have 4.11.1 there or not yet?

Reply:
I guess so, this file is just for devs, the CI does not use it.